src/js/hooks/cycleCount/useCountStepTable.jsx

```diff
@@ -13,6 +13,7 @@ import SelectField from 'components/form-elements/v2/SelectField';
 import TextInput from 'components/form-elements/v2/TextInput';
 import cycleCountColumn from 'consts/cycleCountColumn';
 import { DateFormat } from 'consts/timeFormat';
+import useNavigation from 'hooks/useNavigation';
```

Comment:
useNavigation is a hook that replaced useHistory in the newer react-router-dom version, It would be better to rename this hook in case we want to bump up that library version in the future.